File: C/2029 Reservatório de Mel.c
#include <stdio.h>
#include <math.h>
int main(void) 
{
  double v, d;
  while(scanf("%lf", &v) != EOF)
  {
    scanf("%lf", &d);
    double r = d/2;
    double a = 3.14 * pow(r,2);
    double h = v/a;
    printf("ALTURA = %1.2lf\n", h);
    printf("AREA = %1.2lf\n", a);
  }
  return 0;
}

File: C#/2029 Reservatório de Mel.cs
using System;

class MainClass 
{
  public static void Main (string[] args) 
  {
    Console.Clear();
    string s, s1;
    while(!string.IsNullOrEmpty(s = Console.ReadLine()))
    {
      double v = double.Parse(s);
      double d = double.Parse(Console.ReadLine());
      double r = d/2;
      double a = 3.14 * Math.Pow(r,2);
      double h = v/a;
      Console.WriteLine($"ALTURA = {h:0.00}");
      Console.WriteLine($"AREA = {a:0.00}");
    }
  }
}
